src/Resources/config/config.yml

```diff
@@ -89,3 +89,6 @@ sylius_rbac:
     custom_sections:
         rbac:
             - sylius_rbac
+
+parameters:
+    root_administrator_email: ''
```

Comment:
I would add it at the top of the file (convention + make it more visible, indicating there is something you can override)